### DISCUSSION TITLE
Bot interaction

### DISCUSSION THREAD

**Discussion by alex-user-go** [xelaj/mtproto#70] [Q&A] [answered]
Hi. I am trying to interact with a bot. I've managed to call `MessagesStartBot`, but how should I get the response? I asume there isn't much difference between talking to a bot or a user. Could anyone give me an example?

**Comment by quenbyako** [comment, marked as answer]
@alex-user-go lol, i didn't even realize that this method exist, lmao.

Answer to your question: you can get answer from bot as any another message: there is no difference between bot chats, private chats, supergroups, **except secret chats**. All of them are `Chat` objects :smiley: 

Also, you don't require to use exact this method: you can also start dialog with bot as always, just to send [`MessagesSendMessage` method](https://core.telegram.org/method/messages.sendMessage). Hope that i understood your problem correctly.
